Presentation.Web/app/models/it-contract/YearSegmentOption.ts

```diff
@@ -40,7 +38,9 @@
                     return this.options.filter(x => x.id === YearSegmentOption.EndOfQuarter)[0];
                 case "2":
                     return this.options.filter(x => x.id === YearSegmentOption.EndOfMonth)[0];
-
+                case undefined:
```

Comment:
her var problemet @JacobMalling 
Af en eller anden grund blev det der felt undefined efter navigation (i stedet for null - ved ikke om modellen reduceres automatisk eller hvad det er). Til gengæld omfattes undefined nu, og så virker det

Reply:
Hmm det har nok været noget inde på $scope.contract hvor den måske er blevet nulstillet i et af de andre views? Det er jo $scope.contract der dottes ind på i det kode der kalder det her